hr_edl/src/bin/run_cfr_corr_dist.cc

```diff
@@ -0,0 +1,95 @@
+// This file runs standard CFR on a .efg game.
+//
+// The graph from Sec 6.1 of Morrill et al. 2020, Hindsight and Sequential
+// Rationality of Correlated Play (https://arxiv.org/abs/2012.05874) can be
+// reproduced by running this code on efg/extended_shapleys.efg.
+
+#include <memory>
+#include <queue>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "absl/flags/usage.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_join.h"
+#include "open_spiel/algorithms/cfr.h"
+#include "open_spiel/algorithms/corr_dev_builder.h"
+#include "open_spiel/algorithms/corr_dist.h"
+#include "open_spiel/game_transforms/turn_based_simultaneous_game.h"
+#include "open_spiel/spiel.h"
+
+// Environment
+ABSL_FLAG(std::string, efg_file, "efg/extended_shapleys.efg",
+          "The EFG file representing the game.");
+ABSL_FLAG(int32_t, t, 1000, "The number of iterations to run.");
```

Comment:
This flag is unused right now. Want to set the default to `10000000` and replace the limit in the for loop with `iterations`?

Reply:
Done.